### ISSUE TITLE
No video controls on a Cover Block with video background

### ISSUE THREAD

**Issue by maddisondesigns** [WordPress/gutenberg#38975]
### Description

There's no video controls in the Cover Block when it has a video background.

When you're editing the page, this is incredibly annoying as the video just constantly plays over and over again, which in turn, causes the fan on my laptop to go crazy. Editing a page shouldn't be this resource intensive.

At the bare minimum, there should be a Play/Pause control on the Cover Block toolbar to allow you to stop the video from constantly playing when editing the page.

There should also be settings in the settings sidebar, like the Video Block, to specify whether the video should be looped and/or muted on the frontend.

https://user-images.githubusercontent.com/2078245/155089747-9cc4443b-59a5-4ec4-9e0c-4a08df7e6d78.mp4

### Step-by-step reproduction instructions

- Add cover block and select a video to use as the background
- Watch video play over and over without being able to control it or stop it

### Environment info

WP 5.9
Twenty Twenty One
Firefox 97.0.1
macOS 12.1
No Gutenberg plugin, core only

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by courtneyr-dev** [comment]
It could be useful to have a speed control option here as well when a user is creating a Cover block.  Play the video on loop at .5 speed or less.

**Comment by maddisondesigns** [comment]
An option to control the playback speed is a great idea as well.

**Comment by ramonjd** [comment]
Setting `autoPlay` to `false` on the video element in [Cover Block edit.js](https://github.com/WordPress/gutenberg/blob/922976bbe95c4ab4da06cb9d7e5e25d98f1d398c/packages/block-library/src/cover/edit.js#L758) will stop the video, but yes, it would be better to have a control to toggle this in the editor.

We'd also want the autoPlay setting to be reflected on the focal point picker as well:

<img width="1142" alt="Screen Shot 2022-02-23 at 4 14 56 pm" src="https://user-images.githubusercontent.com/6458278/155265170-d2bff421-e76e-4b2f-99d8-399b97d1af6f.png">

That would remove the "annoyance" effect, at least in the editor.

The only question is whether reflect the state in save.js, and therefore the frontend. If so, then maybe it should be a consideration for [background block support](https://github.com/WordPress/gutenberg/discussions/38119).

Otherwise we'd probably have to add more attributes to the Cover Block, which might not be an ideal long-term solution. See the discussion over at https://github.com/WordPress/gutenberg/pull/38784

**Comment by cbirdsong** [comment]
This is also an accessibility problem in the front end. The lack of a play/pause button is a failure of [WCAG 2.1 Success Criterion 2.2.2: Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html). It also doesn't respect prefers-reduced-motion, which should make the video start out paused.

**Comment by SincroAI** [comment]
This should work. It will hide the playback video controls automatically as well you/the user can control the sound (mute, loud, etc.)

HTML
![VideOverlay](https://user-images.githubusercontent.com/126013176/220430872-be458372-f670-450e-af70-7dc1842a9a6e.png)

<div class="videoWrapper">
      	 
       <video controls autoplay muted loop class="fillWidth fadeIn wow collapse in" data-wow-delay="1s" data-repeat="true" data-volume="60" poster="https://yourdomain.com/vid/someimage.jpg" id="video-background" style="width: 100%;">
           <source src="https://yourdomain.com/vid/somevideo.mp4" type="video/mp4">
       </video>
       
</div>

CSS
.videoWrapper > video {
  width: 100%;
  height: 100%;
  object-fit: cover;       <<< This might not work in IE (earlier versions..)
}

Hope this helps.

**Comment by ryanAllMad** [comment]
Curious if anyone has noticed that this problem also creates an issue for [WCAG 1.1.1 Text Alternatives Level A](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) and if adding the Text Tracks setting to the cover block will be part of this fix?

**Comment by tnchuntic** [comment]
Inline with the settings is it possible also to add a lazy for cover video? Currently it loading the video even if the video is not in viewport. It might conflict with with autoplay, not sure :). 

Found some post about lazy loading from google web.dev https://web.dev/lazy-loading-video/.

**Comment by millertchris** [comment]
Looks like we're not able to manage adding a video thumbnail. Google Search Console is flagging this as an issue and claims that the page will not be crawled because of it.
